hr_employee_phone_extension/models/hr_employee.py

```diff
@@ -23,7 +23,7 @@
 from openerp.osv import fields, orm
 
 
-class hr_employee(orm.Model):
```

Comment:
Can you migrate to new the API? This is a trivial module, so I don't think it is too much to ask.